test/functional/core/startup_spec.lua

```diff
@@ -224,6 +224,23 @@ describe('startup', function()
     end
   end)
 
+  it('-e sets ex mode', function()
+    local screen = Screen.new(25, 3)
+    clear('-e')
+    screen:attach()
+    -- Verify we set the proper mode both before and after :vi.
+    feed("put =mode(1)<CR>vi<CR>:put =mode(1)<CR>")
+    screen:expect{[[
+      cv                       |
+      ^n                        |
+      :put =mode(1)            |
+    ]]}
```

Comment:
```suggestion
    screen:expect([[
      cv                       |
      ^n                        |
      :put =mode(1)            |
    ]])
```

Reply:
Thanks.  I've squashed that into the commit.